app/scripts/modules/core/src/application/modal/PermissionsConfigurer.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
+import * as Select from 'react-select';
+import {Button} from 'react-bootstrap';
+import {ReactInjector} from 'core/reactShims';
```

Comment:
`import {} from 'core/...'` goes after all the external imports with a space between them all.